src/scripts/authcheck.ts

```
@@ -1,18 +1,17 @@
-const hashResult = hash.split('&').reduce(
-  (previous: { [key: string]: unknown }, item) => ({
-    ...previous,
-    [item.split('=')[0]]: item.split('=')[1],
-  }),
-  {},
-);
+const hashResult = hash
+  .split('&')
+  .reduce(
+    (previous: Record<string, unknown>, item) => ({
+      ...previous,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)